pool/session.go

```diff
@@ -49,13 +48,8 @@ func initSession(ctx context.Context, c *sdkClientWrapper, dur uint64, signer us
 		return dst, fmt.Errorf("invalid session token ID: %w", err)
 	}
 
-	var key neofsecdsa.PublicKey
-	if err = key.Decode(res.PublicKey()); err != nil {
```

Comment:
Actually this suggests another inconsistency (SDK-level), sometimes keys are proper keys and sometimes they are bytes. Depends on type.